packages/trader/src/AppV2/Hooks/useOrderDetails.ts

```diff
@@ -108,7 +108,7 @@ const transformAccumulatorData = (data: TContractInfo) => {
         ...{
             ...(data.limit_order?.take_profit && {
                 [CARD_LABELS.TAKE_PROFIT]: data.limit_order?.take_profit?.order_amount
-                    ? `${data.limit_order.take_profit.order_amount} ${data.currency}`
+                    ? `${data.limit_order.take_profit.order_amount.toFixed(2)} ${data.currency}`
```

Comment:
I think the amount of decimal points should be variable based on currency, not fixed to a specific number

Reply:
yup , I am using FormatUtils.formatMoney function to handle the decimal places now . Thanks

Reply:
This is also changed into formatMoney function from shared package